plugins/gateway/kong/plugin.go

```diff
@@ -147,15 +144,19 @@ func createTelemetry(kong *pdk.PDK) (*models.Telemetry, error) {
 	return &telemetry, nil
 }
 
-func parseHost(kongHost string) string {
+// KongHost: <namespace>.<svc-name>.svc.cluster.local.80
```

Comment:
the format was changed? I remember seeing something else

Reply:
fixed